packages/coding-standards/.phpcs.xml.dist-sample

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<ruleset name="Jetpack-Plugin">
+	<config name="minimum_supported_wp_version" value="5.3" />
+	<config name="testVersion" value="5.6-"/>
```

Comment:
Since PHPCompatibility isn't included here, this is redundant. Probably best to keep this config at the point where PHPCompatibility ruleset is included.

Reply:
e64898095